src/coremods/core_user/cmd_mode.cpp

```diff
@@ -45,7 +45,10 @@ CmdResult CommandMode::Handle(const std::vector<std::string>& parameters, User*
 
 	if ((!targetchannel) && (!targetuser))
 	{
-		user->WriteNumeric(Numerics::NoSuchNick(target));
+		if (target[0] == '#')
```

Comment:
Can you use IsChannel here? It will be more future proof for when we have multiple channel types.